frontend/src/pages/Home/home.jsx

```diff
@@ -1,13 +1,256 @@
 import './home.css';
+import Button from '../../components/Button/button';
+import { Link, useNavigate } from 'react-router-dom';
+import ParticipantCard from '../../components/ParticipantCard/participantCard';
+import { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * A reusable section component for the home page.
+ * Renders its children inside a colored section with a container.
+ *
+ * @component
+ * @param {Object} props - Component props.
+ * @param {'white' | 'green' | 'yellow'} props.color - The color theme for the section.
+ * Applied as part of the section's CSS class name.
+ * @param {string} [props.className] - Optional additional CSS classes for the container.
+ * @param {React.ReactNode} props.children - The content to be rendered inside the section.
+ *
+ * @example
+ * <HomeSection color="blue" className="custom-class">
+ *   <h2>Welcome</h2>
+ *   <p>This is the home page.</p>
+ * </HomeSection>
+ */
+function HomeSection({ color, className, children }) {
+    return (
+        <section className={`home--section__${color}`}>
+            <div className={`container home--container ${className}`}>
```

Comment:
If `className` is undefined, this will result in an unwanted "undefined" in the class list. Use:
```
<div className={`container home--container${className ? ' ' + className : ''}`}>
```